tidb-cloud/backup-and-restore.md

```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature.
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
```

Comment:
"nearly eliminate" a little strange to me, what about"Minimize the impact on cluster performance during backup, with the potential to keep the impact on QPS and transaction latency under 1%, while not utilizing any cluster CPU or memory."?